configs/dev-aws-config.yaml

```diff
@@ -0,0 +1,79 @@
+# Configuration options for dev
+tornado:
+  port: 8888
+  debug: true # will reload server on each change to python files
+  xsrf: true # need to make sure frontend is set up to work properly with this setting on
+
+postgres:
+  hostname: localhost
```

Comment:
This should be some IP address or does the aws deployment also use localhost?

Reply:
For the actual AWS EC2 instance, this is indeed the IP address for the database, but because this config is meant to be run locally, the Postgres instance you are developing on is still local.

Reply:
Ahh I see